src/test/java/com/cgdp/library/controller/AuthorControllerTest.java

```diff
@@ -58,16 +60,17 @@ public void should_create_author() throws Exception {
         resultActions
               .andExpect(status().isCreated())
               .andExpect(jsonPath("$.id", is(id.intValue())))
-              .andExpect(jsonPath("$.name", is(authorName)));
+              .andExpect(jsonPath("$.authorName", is(authorName)));
     }
 
     @Test
     public void should_return_bad_request_when_author_name_is_empty() throws Exception {
         // given
-        String authorName = "";
+        String authorName = "John Doe";
         CreateAuthorRequestDTO createAuthorRequestDTO = new CreateAuthorRequestDTO(authorName);
 
-        given(authorService.createAuthor(authorName)).willThrow(new RuntimeException());
+        given(authorService.createAuthor(authorName)).willThrow(new ResponseStatusException(
+              HttpStatus.BAD_REQUEST));
```

Comment:
I believe this can be removed as exception will be thrown before it can reach the service